src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Requests/RealizeAndMeasureUnplannedBuildingRequest.cs

```diff
@@ -0,0 +1,92 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.Building.Requests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using BuildingRegistry.Building;
+    using BuildingRegistry.Building.Commands;
+    using BuildingRegistry.Building.Datastructures;
+    using Newtonsoft.Json;
+    using NodaTime;
+    using Swashbuckle.AspNetCore.Filters;
+
+    [DataContract(Name = "StelGebouwVast", Namespace = "")]
+    public sealed class RealizeAndMeasureUnplannedBuildingRequest
+    {
+        /// <summary>
+        /// De inmeting van het gebouw in GML-3 formaat met Lambert 72 referentie systeem.
```

Comment:
Do we need summaries on every property? Otherwise remove the single summary.

Reply:
I don't think we need the summary here as this action will not be used/displayed in public api.

Reply:
Removed.